data/json/items/comestibles/raw_veggy.json

```diff
@@ -443,6 +443,7 @@
     "charges": 12,
     "bashing": 2,
     "fun": -1,
+    "flags": [ "SMOKABLE" ],
```

Comment:
Needs a "smoking_result"

Reply:
It worked without it for whatever reason, but it's probably better to be explicit.